src/lib/backpack-api.js

```diff
@@ -0,0 +1,19 @@
+const getBackpackContents = ({
+    host,
+    username,
+    token,
+    limit,
+    offset
+}) => fetch(`${host}${username}?limit=${limit}&offset=${offset}`, {
+    headers: {'x-token': token}
+})
+    .then(d => d.json())
```

Comment:
Not sure if/how we should handle the error here, or just raise it up to the component and let the component do it.